### PR TITLE
Issue #DP-951 Task:A user can be contributor and reviewer to same project

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html [hunk 1]
```diff
@@ -33,34 +33,29 @@
...
+            <dl *ngIf="hasAccessFor('showRejected')" class="sb-meta-data">
               <dt>{{countData['reject']}}</dt>
               <dd>Rejected</dd>
             </dl>
-            <dl *ngIf="actions.showUnderReview.roles.includes(sessionContext.currentRoleId)" class="sb-meta-data">
+            <dl *ngIf="hasAccessFor('showUnderReview')" class="sb-meta-data">
               <dt>{{countData['review']}}</dt>
               <dd>Review in Progress</dd>
             </dl>
-            <dl *ngIf="actions.showTotalUnderReview.roles.includes(sessionContext.currentRoleId)"
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:58]: why is this removed?

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts [hunk 2]
```diff
@@ -487,8 +487,7 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
 
   getContentStatusCount(data) {
     const self = this;
-    if (this.sessionContext.currentOrgRole === 'admin' ||
-    (this.sessionContext.currentOrgRole === 'user' && this.sessionContext.currentRole === 'REVIEWER')) {
+    if (['admin', 'user'].includes(this.sessionContext.currentOrgRole)  && this.sessionContext.currentRoles.includes('REVIEWER')) {
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:490]: what is currentOrgRole and currentRoles?

**Comment by twsvaishali** [comment]
@niteshkesarkar we will need code walk-through as there are many files

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html (hunk 1 above)
Comment (niteshpk) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:58]: The total is already shown above this is duplicate so removed.

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:58]: The total is already shown. 
`hasAccessFor('showTotalContribution') || hasAccessFor('showTotalUnderReview')"`

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts (hunk 2 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:490]: currentOrgRole = `string` which can have value -`admin`, `user`, `REVIEWER`, `individual`
currentRole = now, `array` of  `["CONTRIBUTOR"]`, `["REVIEWER"]`, `["REVIEWER", "CONTRIBUTOR"]`, earlier string which can have value - `REVIEWER`, `CONTRIBUTOR`

**Comment by twsvaishali** [comment]
@niteshkesarkar package-lock.json is there in the commit.. please remove
